src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CreateTransactionEOF1InvalidOpcodesFiller.yml

```diff
@@ -0,0 +1,278 @@
+# CreateTransaction: 
+#   - Test all invalid opcodes are not deployed in EOF 
+#   - Test all invalid opcodes makes initcode invalid
+---
+CreateTransactionEOF1InvalidOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      # Invalid opcodes in deployed code
```

Comment:
I would be fine with testing instruction validation only in initcode and removing this entire section.

Reply:
Actually maybe the simplest combination would be legacy initcode creating EOF code - we could leave only it too.